dev/Scroller/TestUI/ScrollerDynamicPage.xaml

```diff
@@ -556,22 +562,45 @@
                     </StackPanel>
 
                     <TextBlock Text="Snaps Points" Margin="2,4,2,2" Grid.Row="27" Grid.ColumnSpan="2"/>
-                    <TextBlock Text="Value:" Grid.Column="0" Grid.Row="28" VerticalAlignment="Center"/>
-                    <TextBox x:Name="txtSnapPointValue" Grid.Column="1" Grid.Row="28" VerticalAlignment="Center"/>
-                    <!--
-                    <TextBlock Text="ApplicableValueRange:" Grid.Column="0" Grid.Row="29" VerticalAlignment="Center"/>
-                    <TextBox x:Name="txtSnapPointValueRange" Text="0, 100000" Grid.Column="1" Grid.Row="29" VerticalAlignment="Center"/>
-                    -->
-                    <TextBlock Text="SnapPointKind:" Grid.Column="0" Grid.Row="30" VerticalAlignment="Center"/>
-                    <ComboBox x:Name="cmbSnapPointKind" Grid.Column="1" Grid.Row="30" VerticalAlignment="Center" SelectedIndex="0">
+
+                    <TextBlock Text="SnapPointKind:" Grid.Column="0" Grid.Row="28" VerticalAlignment="Center"/>
+                    <ComboBox x:Name="cmbSnapPointKind" Grid.Column="1" Grid.Row="28" HorizontalAlignment="Stretch" VerticalAlignment="Center" SelectedIndex="0"
+                        SelectionChanged="CmbSnapPointKind_SelectionChanged">
                         <ComboBoxItem>Vertical</ComboBoxItem>
                         <ComboBoxItem>Horizontal</ComboBoxItem>
                         <ComboBoxItem>Zoom</ComboBoxItem>
                     </ComboBox>
-                    <StackPanel Orientation="Horizontal" Grid.ColumnSpan="2" Grid.Row="31">
-                        <Button x:Name="btnAddSnapPoint" Content="Add Snap Point" Margin="2" Click="BtnAddSnapPoint_Click"/>
-                        <Button x:Name="btnClearSnapPoints" Content="Clear Snap Points" Margin="2" Click="BtnClearSnapPoints_Click"/>
-                    </StackPanel>
+
+                    <Button x:Name="btnClearSnapPoints" Content="Clear Snap Points" HorizontalAlignment="Stretch"
+                        Grid.Row="29" Grid.ColumnSpan="2" Margin="2" Click="BtnClearSnapPoints_Click"/>
+
+                    <TextBlock Text="Alignment:" Grid.Column="0" Grid.Row="30" VerticalAlignment="Center" Margin="0,2,0,0"/>
+                    <ComboBox x:Name="cmbSnapPointAlignment" HorizontalAlignment="Stretch" VerticalAlignment="Center" SelectedIndex="0"
+                        Grid.Column="1" Grid.Row="30" >
+                        <ComboBoxItem>Near</ComboBoxItem>
+                        <ComboBoxItem>Center</ComboBoxItem>
+                        <ComboBoxItem>Far</ComboBoxItem>
+                    </ComboBox>
+
+                    <TextBlock Text="Value:" Grid.Column="0" Grid.Row="31" VerticalAlignment="Center" Margin="0,2,0,0"/>
+                    <TextBox x:Name="txtIrregularSnapPointValue" Grid.Column="1" Grid.Row="31" VerticalAlignment="Center" Margin="0,2,0,0"/>
+                    <!--
```

Comment:
Can delete?

Reply:
We keep those for when we address the "#ifdef ApplicableRangeType" commented-out code.

Reply:
Cool. Make sure that's commented clearly and leave it in.